test/unit/util.test.js

```diff
@@ -2031,6 +2031,238 @@ describe('SCHEMA UTILITY FUNCTION TESTS ', function () {
+                  },
+                  'state': {
+                    'type': 'string',
+                    'example': 'Lisboa'
+                  },
+                  'postalCode': {
+                    'type': 'string',
+                    'example': '1000-427'
+                  },
+                  'countryCode': {
+                    'type': 'integer',
+                    'format': 'int64',
+                    'example': 620
+                  }
+                }
+              }
+            },
+            'parameters': {
+              'contentType': {
+                'name': 'Content-Type',
+                'in': 'header',
+                'schema': {
+                  'type': 'string',
+                  'example': 'application/json'
+                }
+              },
+              'organizationID': {
+                'name': 'organizationID',
+                'in': 'path',
+                'description': 'The id of the organization',
+                'required': true,
+                'example': 8888,
+                'schema': {
+                  'type': 'integer',
+                  'example': 8888
+                }
+              }
+            }
+          },
+          'paths': {
+            '/organization/{organizationID}': {
+              'get': {
+                'summary': 'Get Organization by ID',
+                'operationId': 'getOrganizationByID',
+                'tags': [
+                  'Organization Endpoints'
+                ],
+                'parameters': [
+                  {
+                    'name': 'Content-Type',
+                    'in': 'header',
+                    'schema': {
+                      'type': 'string',
+                      'example': 'application/json'
+                    }
+                  },
+                  {
+                    'name': 'organizationID',
+                    'in': 'path',
+                    'description': 'The id of the organization',
+                    'required': true,
+                    'example': 8888,
+                    'schema': {
+                      'type': 'integer',
+                      'example': 8888
+                    }
+                  }
+                ],
+                'responses': {
+                  '200': {
+                    'description': 'The required organization detail',
+                    'content': {
+                      'application/json': {
+                        'schema': {
+                          '$ref': '#/components/schemas/Organization'
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        pmResponse = SchemaUtils.convertToPmResponse(response, '200', originalRequest, components).toJSON(),
+        pathVariables = _.get(pmResponse, 'originalRequest.url.variable');
+
+      expect(JSON.parse(pmResponse.body)).to.eql(
```

Comment:
@Akshay2402 Ideally we shouldn't include other tests in this test as we are only checking for certain variables to be present in the request of an example. We can create another test case for it.

Reply:
Sure will update the test accordingly 🙇‍♂️